### PR TITLE
Adds a variety of spacing fixes

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

File: lua/autorun/client/cl_contact_forms.lua [hunk 1]
```diff
@@ -49,7 +48,7 @@ local function processFieldsForForm( fields, formData )
         for _, fieldStruct in pairs( fields ) do
             local field = fieldStruct.field
 
-            print("Sending '" .. fieldStruct.name .. "' to the server..")
+            print( "Sending '" .. fieldStruct.name .. "' to the server .. " )
```
Comment (overlordbleck) [inline lua/autorun/client/cl_contact_forms.lua]:
```suggestion
            print( "Sending '" .. fieldStruct.name .. "' to the server..." )
```

File: lua/autorun/client/cl_contact_forms.lua [hunk 2]
```diff
@@ -172,7 +171,7 @@ local function makeBooleanField( question, parent )
         YesButton:SetImage( formImage( "radio-filled" ) )
         NoButton:SetImage( formImage( "radio" ) )
     end
-    ---
+    -- -
```
Comment (overlordbleck) [inline lua/autorun/client/cl_contact_forms.lua]:
```suggestion
    ---
```

File: lua/autorun/client/cl_contact_forms.lua [hunk 3]
```diff
@@ -186,7 +185,7 @@ local function makeBooleanField( question, parent )
         YesLabel:SetFontInternal( "Trebuchet24" )
         YesLabel:SetToFullHeight()
     end
-    ---
+    -- -
```
Comment (overlordbleck) [inline lua/autorun/client/cl_contact_forms.lua]:
```suggestion
    ---
```

File: lua/autorun/client/cl_contact_forms.lua [hunk 4]
```diff
@@ -198,7 +197,7 @@ local function makeBooleanField( question, parent )
         NoButton:SetImage( formImage( "radio-filled" ) )
         YesButton:SetImage( formImage( "radio" ) )
     end
-    ---
+    -- -
```
Comment (overlordbleck) [inline lua/autorun/client/cl_contact_forms.lua]:
```suggestion
    ---
```

File: lua/autorun/client/cl_contact_forms.lua [hunk 5]
```diff
@@ -212,7 +211,7 @@ local function makeBooleanField( question, parent )
         NoLabel:SetFontInternal( "Trebuchet24" )
         NoLabel:SetToFullHeight()
     end
-    ---
+    -- -
```
Comment (overlordbleck) [inline lua/autorun/client/cl_contact_forms.lua]:
```suggestion
    ---
```

File: lua/autorun/server/sv_cfc_contact_forms.lua [hunk 6]
```diff
@@ -3,21 +3,21 @@ util.AddNetworkString( 'CFC_SubmitFeedbackForm' )
 util.AddNetworkString( 'CFC_SubmitBugReport' )
 util.AddNetworkString( 'CFC_SubmitPlayerReport' )
 
-local FORM_PROCESSOR_URL = file.Read( "cfc/contact/url.txt", "DATA" )
-FORM_PROCESSOR_URL = string.Replace(FORM_PROCESSOR_URL, "\r", "")
-FORM_PROCESSOR_URL = string.Replace(FORM_PROCESSOR_URL, "\n", "")
+local FORM_PROCESSOR_URL = file.Read( 'cfc/contact/url.txt', 'DATA' )
+FORM_PROCESSOR_URL = string.Replace( FORM_PROCESSOR_URL, '\r', '' )
+FORM_PROCESSOR_URL = string.Replace( FORM_PROCESSOR_URL, '\n', '' )
```
Comment (overlordbleck) [inline lua/autorun/server/sv_cfc_contact_forms.lua:8]: Why are we preferring single quotations?

**Review by overlordbleck — COMMENTED** [review]
Assuming that most of these changes are by Jenkins, I think we should find a way for him to not change stuff in comments or strings.

File: lua/autorun/server/sv_cfc_contact_forms.lua (hunk 6 above)
Comment (sarahsturgeon) [reply lua/autorun/server/sv_cfc_contact_forms.lua:8]: The only lua guide about quotes is "use one or the other, but not both". in this file we used singles more than we used doubles so it was easier to change the doubles to singles

**Comment by sarahsturgeon** [comment]
Closing in favor of: https://github.com/CFC-Servers/cfc_contact_forms/pull/5

**Pull request closed without merging**